src/current/_includes/releases/v24.1/v24.1.2.md

```diff
@@ -0,0 +1,71 @@
+## v24.1.2
+
+Release Date: July 2, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-1-2-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- Fixed a bug in CockroachDB v22.2 and later where long running initial scans may incorrectly restore [checkpoint job](https://www.cockroachlabs.com/docs/v24.1/how-does-an-enterprise-changefeed-work) progress and drop events during node (and thus [changefeed](https://www.cockroachlabs.com/docs/v24.1/create-changefeed)) restart. This issue was most likely to occur in clusters where (1) the `changefeed.shutdown_checkpoint.enabled` [cluster setting](https://www.cockroachlabs.com/docs/v24.1/cluster-settings) is `true`, (2) there are multiple table targets in a changefeed, or (3) low values of the cluster settings `changefeed.frontier_checkpoint_frequency` or [`changefeed.frontier_highwater_lag_checkpoint_threshold`](https://www.cockroachlabs.com/docs/v24.1/cluster-settings#setting-changefeed-frontier-highwater-lag-checkpoint-threshold) are in effect. [#124996][#124996]
+- Added an error message for unsupported Apache Pulsar [changefeed](https://www.cockroachlabs.com/docs/v24.1/create-changefeed) sink parameters. [#124666][#124666]
+
+<h3 id="v24-1-2-operational-changes">Operational changes</h3>
+
+- Improved [disk usage metric reporting](https://www.cockroachlabs.com/docs/v24.1/ui-cluster-overview-page#capacity-metrics) over volumes that dynamically change their size over the life of the `cockroach` process. [#125050][#125050]
+- The following [cluster settings](https://www.cockroachlabs.com/docs/v24.1/cluster-settings) have had their default values updated from `100000` to `7500`. These settings control the maximum number of [fingerprints](https://www.cockroachlabs.com/docs/v24.1/ui-statements-page#sql-statement-fingerprints) (distinct combinations of statements and [transactions](https://www.cockroachlabs.com/docs/v24.1/transactions)) we can store in memory, with stats being flushed on a `10m` [interval](https://www.cockroachlabs.com/docs/v24.1/interval) by default. Users with workloads producing unique fingerprints higher than this amount within the flush interval may notice that they are missing SQL stats. Users who want to capture those missing stats may increase these settings as necessary. [#125554][#125554]
```

Comment:
```suggestion
- The default values for the following [cluster settings](https://www.cockroachlabs.com/docs/v24.1/cluster-settings) were updated from `100000` to `7500`. These settings control the maximum number of [fingerprints](https://www.cockroachlabs.com/docs/v24.1/ui-statements-page#sql-statement-fingerprints) (distinct combinations of statements or [transactions](https://www.cockroachlabs.com/docs/v24.1/transactions)) CockroachDB can store in memory, with stats being flushed on a `10m` [interval](https://www.cockroachlabs.com/docs/v24.1/interval) by default. You can increase these settings if your workload produces more unique fingerprints than this amount within the flush interval, and you notice that SQL stats are missing. [#125554][#125554]
```

Reply:
Thanks! I have slurped in this whole thing except that the release note commit message in cockroachdb/cockroach#125554 said "statements _and_ transactions" so I'm leaving that term